### ISSUE TITLE
Error with image picker

### ISSUE THREAD

**Issue by romaluca** [markmooibroek/apn_fb_login#2]
Hi, i'm using your plugin in my app.
For login in it works well.
The issue is that i use also image_picker plugin.
And when use it for picking an image i receive this error:
```

E/AndroidRuntime( 4625): java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=2342, result=-1, data=Intent { (has extras) }} to activity {com.shuttertop.app/com.shuttertop.app.MainActivity}: java.lang.NullPointerException: Attempt to invoke interface method 'boolean com.facebook.CallbackManager.onActivityResult(int, int, android.content.Intent)' on a null object reference
E/AndroidRuntime( 4625): 	at android.app.ActivityThread.deliverResults(ActivityThread.java:4324)
E/AndroidRuntime( 4625): 	at android.app.ActivityThread.handleSendResult(ActivityThread.java:4367)
E/AndroidRuntime( 4625): 	at android.app.ActivityThread.-wrap19(Unknown Source:0)
E/AndroidRuntime( 4625): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1649)
E/AndroidRuntime( 4625): 	at android.os.Handler.dispatchMessage(Handler.java:105)
E/AndroidRuntime( 4625): 	at android.os.Looper.loop(Looper.java:164)
E/AndroidRuntime( 4625): 	at android.app.ActivityThread.main(ActivityThread.java:6541)
E/AndroidRuntime( 4625): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime( 4625): 	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
E/AndroidRuntime( 4625): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
E/AndroidRuntime( 4625): Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'boolean com.facebook.CallbackManager.onActivityResult(int, int, android.content.Intent)' on a null object reference
E/AndroidRuntime( 4625): 	at com.appnormal.plugin.fblogin.ApnFbLoginPlugin.onActivityResult(ApnFbLoginPlugin.java:133)
E/AndroidRuntime( 4625): 	at io.flutter.app.FlutterPluginRegistry.onActivityResult(FlutterPluginRegistry.java:160)
E/AndroidRuntime( 4625): 	at io.flutter.app.FlutterActivityDelegate.onActivityResult(FlutterActivityDelegate.java:132)
E/AndroidRuntime( 4625): 	at io.flutter.app.FlutterActivity.onActivityResult(FlutterActivity.java:122)
E/AndroidRuntime( 4625): 	at android.app.Activity.dispatchActivityResult(Activity.java:7235)
E/AndroidRuntime( 4625): 	at android.app.ActivityThread.deliverResults(ActivityThread.java:4320)
E/AndroidRuntime( 4625): 	... 9 more

```

**Comment by markmooibroek** [comment]
Thanks for bringing it to my attention, ill have a look this weekend!

**Comment by markmooibroek** [comment]
Fixed in 0.0.3 :-)

**Comment by romaluca** [comment]
thanks!
